### DISCUSSION TITLE
The ability to duplicate configs

### DISCUSSION THREAD

**Discussion by mmikkel** [craftcms/ckeditor#110] [Ideas] [answered]
Sometimes you want to create a new CKEditor config that is very similar to an existing config, but with a small change or two.

With Redactor, we were exclusively using the file-based JSON configs, so duplicating an existing config and tweaking it was very simple. 

While copying an existing CKEditor config's YAML file and changing its UID is doable, it'd be nice if it was possible to duplicate (or, "Save as new") an existing CKEditor config via the control panel.

**Comment by brandonkelly** [comment, marked as answer]
Added for the next release! (48011e8a8bc805b57368e671d02015f2dd2ac2b7)

**Reply by brandonkelly** [reply]
CKEditor 3.5.0 is out with this :tada:

Note that the update requires [Craft 4.5.0](https://craftcms.com/blog/craft-4-5-released).
